lib/migration.js

```diff
@@ -387,6 +387,33 @@ function mixinMigration(PostgreSQL) {
     return colDefault ? (' DEFAULT ' + colDefault) : '';
   };
 
+  // override this function from base connector to allow postgres connector to
+  // accept dataPrecision and dataScale as column specific properties
+  PostgreSQL.prototype.columnDataType = function(model, property) {
+    var columnMetadata = this.columnMetadata(model, property);
+    var colType = columnMetadata && columnMetadata.dataType;
+    if (colType) {
+      colType = colType.toUpperCase();
+    }
+    var prop = this.getModelDefinition(model).properties[property];
+    if (!prop) {
+      return null;
+    }
+    var colLength = columnMetadata && columnMetadata.dataLength || prop.length || prop.limit;
+    var colPrecision = columnMetadata && columnMetadata.dataPrecision;
+    var colScale = columnMetadata && columnMetadata.dataScale;
+    // info on setting column specific properties
+    // i.e dataLength, dataPrecision, dataScale
+    // https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html
+    if (colType && colLength)
```

Comment:
I would refactor this since `colType` is a common condition: 
```
if(colType) {
if (colLength) {
// return ..
}
if ( colPrecision && colScale) {
// return ...
}
}